### DISCUSSION TITLE
Lineup "select match" grayed out

### DISCUSSION THREAD

**Discussion by amif2000** [ho-dev/HattrickOrganizer#1294] [Q&A]
I'm probably doing something wrong but the "select match" button in the lineup tab is grayed out. This prevents me from uploading my lineup to HT.
My next matches show in the matches tab. I have supporter and HO is approved to set match orders via CHPP.

**Comment by Achill-3s** [comment]
Same issue, was it fixed?

**Reply by amif2000** [reply]
Not a fix, but downgrading Java from 17 to 15 allows this feature to work once per invocation of the program

**Reply by Achill-3s** [reply]
How can you pick which JRE version executes with HO program?
Do i need to uninstall all java version and install version 15 again or is there an easier way?

**Comment by amif2000** [comment]
I downloaded the portable (as in not installed) version with its own JRE.

**Reply by Achill-3s** [reply]
Tried to run the portable edition after uninstalling the normal stable version.
Still same issue.
I even uninstalled other java versions and left the portable JRE just to be sure.
Matches upload/ download is till greyed out.

**Comment by Achill-3s** [comment]
Thanks Man,
I installed H.O 4.1 Portable and it works fine for Download/Upload LineUps.
https://github.com/akasolace/HO/releases/download/4.1/HO-4.1.3310.2-portable-win-JRE.zip

Neither HO 5.1 or 6.0 seems to be working for me, either portable or stable version for Lineups Uploading.
Both versions are greyed out, no matter which JRE is used for executing, thus cannot set match orders via HO on them.

**Comment by akasolace** [comment]
@Achill-3s  as we can't reproduce, can you share your db?

**Reply by Achill-3s** [reply]
Sure, when you say db, i assume the db folder inside the portable HO 5.1 where the lineup upload/download don't work.
I create attachment with only db folder and another attachment with whole portable 5.1 including db.

Whole Portable Folder:
---Removed---

Only DB Folder:
---Removed---

Some Hrf In case you need them:
---Removed---

Portable 5.1 Issue
![image](https://user-images.githubusercontent.com/98124706/150761610-6c1a7fb2-be35-4fdd-92da-f107313c51d7.png)

Portable 4.1 Works fine.
![image](https://user-images.githubusercontent.com/98124706/150761838-e5472d8b-2137-43bd-8a15-7ee1a4724f65.png)

**Comment by akasolace** [comment]
@amif2000  yes please share you db with the service of your choice and send me by ht-mail for privacy ....
@wsbrenk I guess you havent looked into it yet, have you recieved similar comment ?

**Reply by amif2000** [reply]
@akasolace I sent you a link to the database in your private gmail. Not sure if you got it.

**Reply by amif2000** [reply]
Just saw that the 5.2 beta fixes the issue. I downloaded and indeed it works. Happy!
